spec.html

```diff
@@ -21105,12 +21115,12 @@ <h2>Syntax</h2>
         Declaration[?Yield, ?Await]
     </emu-grammar>
 
-    <emu-clause id="sec-block-static-semantics-early-errors">
+    <emu-clause id="sec-block-static-semantics-early-errors" oldids="sec-block-duplicates-allowed-static-semantics">
       <h1>Static Semantics: Early Errors</h1>
       <emu-grammar>Block : `{` StatementList `}`</emu-grammar>
       <ul>
         <li>
-          It is a Syntax Error if the LexicallyDeclaredNames of |StatementList| contains any duplicate entries.
+          It is a Syntax Error if the LexicallyDeclaredNames of |StatementList| contains any duplicate entries<span normative-optional>, unless IsStrict(this production) is *false*, the duplicate entries are only bound by FunctionDeclarations, and the host is a web browser or otherwise supports <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics" title></emu-xref></span>.
         </li>
```

Comment:
This renders poorly, and in particular misses the "normative optional" text: https://ci.tc39.es/preview/tc39/ecma262/sha/06e2af4a47bd749e5cbb0fb0c737221e86685a85/multipage/ecmascript-language-statements-and-declarations.html#sec-block-static-semantics-early-errors

I wonder if moving the attribute up a level would improve things.
```suggestion
        <li>
          <p>It is a Syntax Error if the LexicallyDeclaredNames of |StatementList| contains any duplicate entries.</p>
          <p normative-optional>If the host is a web browser or otherwise supports <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics" title></emu-xref>, the above Syntax Error does not apply when IsStrict(this production) is *false* and the duplicate entries are only bound by FunctionDeclarations.</p>
        </li>
```

Reply:
> This renders poorly, and in particular misses the "normative optional" text:

I agree that it renders poorly, but I'm not sure what you mean by "misses". The normative optional text is there, but the bottom of each line is slightly obscured/clipped by the padding of the line below.

ecmarkup.css probably needs a new rule for when `normative-optional` occurs on inline content. (Currently, that attribute is only attached to block content, specifically `emu-clause` elements.) E.g., if you reduce the padding from 0.5em to 0.2em, the clipping goes away. (Also, the border-left line maybe doesn't make sense for inline content.)

Reply:
> I agree that it renders poorly, but I'm not sure what you mean by "misses". The normative optional text is there, but the bottom of each line is slightly obscured/clipped by the padding of the line below.

I mean that the literal text "NORMATIVE OPTIONAL" is absent, but [Conformance](https://tc39.es/ecma262/multipage/conformance.html#sec-conformance) indicates that it should be present:
> A Normative Optional clause is denoted in this specification with the words "Normative Optional" in a coloured box, as shown below.

<details><summary>current</summary>

![screenshot](https://github.com/tc39/ecma262/assets/1199584/38067914-1009-423a-8cef-418403283484)

</details>
<details><summary>proposed</summary>

![screenshot](https://github.com/tc39/ecma262/assets/1199584/960a31cd-41d4-4145-8445-df7ea697d91b)

</details>

Reply:
> I mean that the literal text "NORMATIVE OPTIONAL" is absent

Ahh, gotcha.